### PR TITLE
Artie Datadog integration

### PR DESCRIPTION
**Pull request by danafallon**
Note: This PR is pointed at our fork of Datadog's integrations repo, so this is for internal (Artie) review only. When we're ready to submit it to Datadog, I'll open another PR against their repo for them to review it.

### CODE REVIEW

File: artie/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Artie Integration
+
+## Overview
+
+[Artie][1] is a real-time database replication product for reliably and consistently syncing data from source databases to data warehouses.
```
Comment (Tang8330) [inline artie/README.md]: Can we do [Artie](https://www.artie.com)?

File: artie/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+The included dashboard should start to show data within 5-10 minutes of connecting the integration, as long as you have data flowing in your Artie deployments.
+
+## Data Collected
+
+### Metrics
+
+Artie reports the following metrics for each of your deployments:
+
+1. Rows processed - the number of rows synced from your source database to your destination database/data warehouse
```
Comment (Tang8330) [inline artie/README.md]: Can we make this into a tabular format?

**Review by Tang8330 — APPROVED** [review]

File: artie/README.md (hunk 1 above)
Comment (danafallon) [reply artie/README.md]: Instead of what?

Comment (Tang8330) [reply artie/README.md]: Like this instead of the footnotes:

```md
Can we do [Artie](https://www.artie.com)? 
```

Comment (danafallon) [reply artie/README.md]: Oh it's the same in practice, but datadog's style guidelines want links to be formatted like this.

Comment (danafallon) [reply artie/README.md]: Here's how it looks: https://github.com/artie-labs/integrations-extras/blob/bf1b04b1908d3497f6a7133b11506bbcd0813cea/artie/README.md

Comment (Tang8330) [reply artie/README.md]: Oh okay cool

**Pull request closed without merging**